pkg/capture/GPClassify.go

```diff
@@ -13,34 +13,21 @@
 
 package capture
 
+type Direction uint8
+
 // Direction detection states
 const (
-	Unknown uint8 = iota
+	Unknown Direction = iota
 	DirectionRemains
 	DirectionReverts
+	DirectionMaybeRemains
```

Comment:
This is an extension to support coming up with a "preliminary" verdict (which will already categorize the packet / flow, but still trigger further packets to be assessed). Right now it's not used (I thought I wanted to use it but then realized that my idea didn't pan out) but I think it's a quite neat addition for future protocol logic, so I'd like to keep it.